sihl-user/src/sihl_user.ml

```diff
@@ -181,23 +181,25 @@ module Make (Repo : User_repo.Sig) : Sihl.Contract.User.Sig = struct
       else Lwt_result.fail IncorrectPassword
   ;;
 
+  let start () = Lwt.return ()
```

Comment:
Please also do this for other commands (migration command for example).

Reply:
Done

Reply:
Thx!